src/api/demo/pages/scan.tsx

```diff
@@ -1,19 +1,24 @@
 import Head from "next/head";
 import Image from "next/image";
+import { readFile } from "fs/promises";
 import styles from "../styles/Home.module.css";
 
+declare const globalThis: any;
```

Comment:
`globalThis` should always be defined?

Reply:
Hm yep. For some reason this was giving me trouble at some point but seems to work fine without it now.